setup.cfg

```diff
@@ -103,6 +103,7 @@ tests =
     # https://github.com/pytest-dev/pytest/issues/12263
     pytest>=6,<8.2
     towncrier>=24.7.0
+    setuptools>=71.1
```

Comment:
For typing?

Reply:
Yes, it includes some needed type hints for `pkg_resources`, or something

Reply:
```suggestion
    # Library stubs for "pkg_resources":
    setuptools>=71.1
```